contributors.md

```diff
@@ -2,90 +2,107 @@
 title: "Contributors"
 ---
 
-The [code of conduct](conduct.md) governs all forms of participation in the R-multiverse project, including package contributions, issues, discussions, and the development of infrastructure.
-Administrators, moderators, and contributors are all subject to its terms.
+This page explains how to contribute packages to R-multiverse.
 
-## How to register a package with R-multiverse
+## Preparation
 
-### The one-time registration process
+[R-multiverse policies](policies.md) govern all forms of participation in R-multiverse,
+including package contributions.
+Administrators, moderators, and contributors are all subject to the terms therein.
+The [review policy](review.md) specifically governs package contribution and review.
+
+In addition, R-multiverse pulls R packages directly from
+[releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) on GitHub and GitLab.
+Before contributing a package to R-multiverse, please ensure it is available
+in a public GitHub/GitLab source code repository with at least one
+[release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) ([example](https://github.com/pola-rs/r-polars/releases/tag/v0.21.0)).
+
+## Registration
+
+To register a package with R-multiverse:
 
 1. Navigate to <https://github.com/r-multiverse/contributions>.
-2. Open a [GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) to add one or more listings to the [`packages` folder](https://github.com/r-multiverse/contributions/tree/main/packages).
+1. Contribute a [package listing](https://github.com/r-multiverse/contributions/tree/main/packages) via
+[GitHub pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests).
 
 Watch this 1-minute video to learn how:
 
 {{< video https://vimeo.com/923333903 title="R-multiverse contribution tutorial" >}}
```

Comment:
could this video get subtitles?

Reply:
As we discussed today, it's a visual demo, no audio.